src/mqttcm_connect.c

```diff
@@ -788,14 +788,14 @@ void publish_notify_mqtt(char *pub_topic, void *payload, ssize_t len)
 
 	rc = mosquitto_publish_v5(mosq, NULL, pub_topic, len, payload, 2, false, props);
 
-	MqttCMInfo("Publish rc %d\n", rc);
+	MqttCMDebug("Publish rc %d\n", rc);
         if(rc != MOSQ_ERR_SUCCESS)
 	{
                 MqttCMError("Error publishing: %s\n", mosquitto_strerror(rc));
         }
 	else
 	{
-		MqttCMInfo("Publish payload success %d\n", rc);
+		MqttCMDebug("Publish payload success %d\n", rc);
```

Comment:
I would suggest to enable any one log in payload publish flow else we cant track whether msg is received from components to mqttcm